### PR TITLE
Updating the GBD health categories page post meeting

### PR DESCRIPTION
**Pull request by SylLutze**
I think this reflects what we discussed in our last docs session but let me know if anything is still unclear.

### CODE REVIEW

**Review by aflaxman — APPROVED** [review]

File: docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst [hunk 1]
```diff
@@ -49,7 +49,7 @@ In particular, we will examine:
 
 To help with understanding some of the below information, a diagram of the heart failure impairment is provided: 
 
-.. image:: health_state_impairment_sequela.svg
+.. image:: overview_diagram.png
```
Comment (zmbc) [inline docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst:52]: Let's include the Visio file in this repo for easy editing later!

File: docs/source/model_design/vivarium_model_components/GBD_disease_health/cause_hierarchy_ihd_hhd_v2.svg
Comment (zmbc) [inline docs/source/model_design/vivarium_model_components/GBD_disease_health/cause_hierarchy_ihd_hhd_v2.svg:1]: Is this used? Also, what does it mean that the two bottom-left sequelae boxes have dashed borders? I'm realizing now that I'm looking at it closely that I don't understand the "sequelae" row -- how are the sequelae grouped into boxes?

File: docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst [hunk 2]
```diff
@@ -119,11 +121,18 @@ modeling, this wouldn't be possible. Usually, information is calculated at the c
...
-modeling. 
+category called impairments. An impairment is a medical condition that GBD has extra 
+data on, but can't be included in the cause hierarchy since it has more than one 
+cause. Again, deciding what is a cause that can fit in the hierarchy and what is an 
+impairment is often a result of historical context and modeler decisions. 
+
+An easy example to grasp is blindness. There is a lot of data available about blindness, but 
+it can be caused by a lot of different things. Therefore, it is difficult to place in the 
+cause hierarchy, but we want to include all the rich data we have. Hence, it is an 
+impairment.
```
Comment (zmbc) [inline docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst:132]: I think this example is really helpful!

File: docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst [hunk 3]
```diff
@@ -119,11 +121,18 @@ modeling, this wouldn't be possible. Usually, information is calculated at the c
...
+cause. Again, deciding what is a cause that can fit in the hierarchy and what is an 
+impairment is often a result of historical context and modeler decisions. 
+
+An easy example to grasp is blindness. There is a lot of data available about blindness, but 
+it can be caused by a lot of different things. Therefore, it is difficult to place in the 
+cause hierarchy, but we want to include all the rich data we have. Hence, it is an 
+impairment. 
+
+There can even be impairment hierarchies. For example, mild heart failure is an impairment, 
+but it also fits under the larger impairment of heart failure.
```
Comment (zmbc) [inline docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst:135]: I think I'd move this after the next paragraph.

File: docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst [hunk 4]
```diff
@@ -79,7 +79,9 @@ not modeled with the same complexity as causes.
 Sequelae detail a specific medical condition associated with
```
Comment (zmbc) [inline docs/source/model_design/vivarium_model_components/GBD_disease_health/index.rst:79]: Not really related to this PR, but could we add "Sequelae are only used in GBD non-fatal estimates; deaths are attributed to a cause but not to a sequela within that cause." to the previous paragraph? Assuming that sounds correct 😄

**Review by alibow — APPROVED** [review]

**Pull request merged**